src/main/java/hudson/plugins/scm/koji/KojiChangeLogSet.java

```diff
@@ -83,4 +81,60 @@ public User getAuthor() {
 
     }
 
+    private static List<Hyperlink> getListFromString(String string) {
+        List<Hyperlink> hyperlinks = new ArrayList<>();
+        hyperlinks.add(new Hyperlink(string));
+        return hyperlinks;
+    }
+
+    private static List<Hyperlink> getListFromRPMs(List<RPM> rpms) {
+        List<Hyperlink> hyperlinks = rpms.stream().filter(rpm -> rpm.hasUrl()).map(rpm -> new Hyperlink(rpm.toString(), rpm.getUrl())).collect(Collectors.toList());
```

Comment:
- please format this better (like here for example https://stackoverflow.com/a/42153650/1597741 much more readable)
- `rpm -> rpm.hasUrl()` can be method reference
- could be one-liner